### ISSUE TITLE
link error for sub-dir package on github.com repo

### ISSUE THREAD

**Issue by subchen** [gojp/goreportcard#136]
I get a report for `github.com/subchen/goutils/log` package.
https://goreportcard.com/report/github.com/subchen/goutils/log

But all source file links on this report page is wrong.

```
Actual:
    https://github.com/subchen/goutils/blob/master/log/logger.go
Expected:
    https://github.com/subchen/goutils/log/blob/master/logger.go
```

**Comment by shawnps** [comment]
Probably related to this change I made recently https://github.com/gojp/goreportcard/commit/122f58ead56bb81b6559b4ed0bfb9bda7ca3d03a

**Comment by shawnps** [comment]
Looks unrelated to 122f58e actually

**Comment by shawnps** [comment]
Looks like I fixed this in d7ca2b1. Can reopen if that's not the case
